paper-scroll-header-panel.html

```diff
@@ -135,17 +136,28 @@
     #condensedHeaderBg {
       opacity: 0;
     }
+
+    ::content .header-title {
```

Comment:
need to select the parent of the `<content>` element for compatibility with Shady DOM. e.g.
```
#headerContainer ::content .header-title {
```